binderhub/main.py

```diff
@@ -46,6 +53,7 @@ def get(self, provider_prefix, _unescaped_spec):
             raise HTTPError(400, str(e))
 
         provider_spec = f'{provider_prefix}/{spec}'
+        social_desc = ": ".join([provider_prefix, SPEC_NAMES[provider_prefix]])
```

Comment:
This will render to something like `gh: GitHub repository` no? Is that what you meant to do? I was expecting it to render to something like `GitHub repository: <org>/<repo>`

Reply:
oops, yep, will fix